en/setup/dev_env_windows.md

```diff
@@ -1,122 +1,59 @@
 # Windows Installation Instructions
 
-There are a number of ways you can set up a Windows development toolchain for PX4. These allow you to build for NuttX/Pixhawk and jMAVSim simulator targets.
+There are a number of ways you can set up a Windows development toolchain for PX4. Depending on what option you choose these allow you to build for NuttX/Pixhawk and jMAVSim simulator targets.
 
-> **Warning** The Windows toolchains are not officially supported (and we discourage their use). They are unbearably slow during Firmware compilation and do not support new boards like Snapdragon Flight. They also cannot run the standard robotics software packages many developers use to prototype computer vision and navigation.
+> **Warning** The Windows toolchains are not officially supported (and we discourage their use). Depending on the option you choose they might be slow and not support all targets. Some options also cannot run the standard robotics software packages many developers use to prototype computer vision and navigation.
 
 <span></span>
-> **Tip** Before starting to develop on Windows, consider installing a dual-boot environment with [Ubuntu](http://ubuntu.com).
```

Comment:
I like the individual pro/con lists, but I wonder if it might also be worth having an overview of all of them in a table. Headings/fields like Gazebo, Jmavsim, NuttX, Native, Memory/Resources etc? Just an idea.

Reply:
Sounds super useful, it could even be colored red, green, yellow... I'll have a look at how to do that.

Reply:
Try avoid using colour. It is very rarely necessary or done well. The key to great doc is minimal use of highlighting and smart use of structure/headings :-)

Reply:
ok, thanks for the hint, I can learn a lot from you. I'll kill the "coloring project" and make a non-highlighted table.